Makefile

```diff
@@ -17,12 +17,12 @@ fmt:
 audit:
 	go list -m all | nancy sleuth
 .PHONY: lint
-lint:
+lint: fmt
```

Comment:
I would not add fmt to lint target as this runs in ci and therefore can change the code but does not take affect in the binary that is deployed or the code stored in repository.

Can we remove this, it is up to developers to fmt the code **not** ci